### PR TITLE
Add project information

### LINKED ISSUE #333: Add project descriptions
There should be more information on the front page, like a link to the Website (maybe also this repo), and a warning that it's currently in preview and features may change at any time without prior notice (or something like that).

* add information to README
* add information to landing page
* add contributers file

### PR DESCRIPTION
**Pull request by ClFeSc**
Closes #333

EDIT: 

image

image
https://github.com/hpi-sam/digital-fuesim-manv/tree/feature/333-more-information#contributors

### CODE REVIEW

**Comment by ClFeSc** [comment]
@christianzoellner FYI

**Comment by ClFeSc** [comment]
@Dassderdie Could you please take a look at the HTML and improve the layout a bit?

File: frontend/src/app/pages/landing-page/landing-page/landing-page.component.html [hunk 1]
```diff
@@ -80,4 +79,76 @@ <h1>Digitale FüSim MANV</h1>
...
+                >Julian Schmidt</a
+            >, <a href="https://github.com/ClFeSc">Clemens Schielicke</a>,
+            Florian Krummrey, Marvin Müller-Mettnau,
+            <a href="https://hpi.de/giese/people/christian-zoellner.html"
+                >Christian Zöllner</a
+            >
+            und
+            <a href="https://hpi.de/giese/people/matthias-barkowsky.html"
+                >Matthias Barkowsky</a
+            >
```
Comment (christianzoellner) [inline frontend/src/app/pages/landing-page/landing-page/landing-page.component.html:151]: please put me last. in the scientific context, the supervisor / owner always goes last :-)

**Comment by ClFeSc** [comment]
@anonym-HPI @hpistudent72 Do you want some page (e.g. your GitHub-Homepage) linked in the footer?

File: frontend/src/app/pages/landing-page/landing-page/landing-page.component.html [hunk 2]
```diff
@@ -80,4 +79,76 @@ <h1>Digitale FüSim MANV</h1>
...
+                href="https://www.bbk.bund.de/DE/Themen/Akademie-BABZ/akademie-babz_node.html"
+                >Bundesakademie für Bevölkerungsschutz und Zivile
+                Verteidigung</a
+            >
+            am
+            <a href="https://www.bbk.bund.de/"
+                >Bundesamt für Bevölkerungsschutz und Katastrophenhilfe</a
+            >
+            entwickelten Simulation für medizinische Führungskräfte für den Fall
+            eines Massenabfalls an Verletzten (MANV).
```
Comment (anonym-HPI) [inline frontend/src/app/pages/landing-page/landing-page/landing-page.component.html:101]:
```suggestion
            eines Massenanfalls an Verletzten (MANV).
```

File: frontend/src/app/pages/landing-page/landing-page/landing-page.component.html [hunk 3]
```diff
@@ -80,4 +79,76 @@ <h1>Digitale FüSim MANV</h1>
...
+        </p>
+        <p>
+            Der Quelltext dieser Webseite und der dahinterstehenden Technologien
+            steht als Open-Source-Software unter
```
Comment (anonym-HPI) [inline frontend/src/app/pages/landing-page/landing-page/landing-page.component.html:105]:
```suggestion
            Der Quelltext dieser Webseite und die dahinterstehenden Technologien
            stehen als Open-Source-Software unter
```

File: frontend/src/app/pages/landing-page/landing-page/landing-page.component.html [hunk 4]
```diff
@@ -80,4 +79,76 @@ <h1>Digitale FüSim MANV</h1>
...
+            <a href="https://github.com/hpi-sam/digital-fuesim-manv"
+                >https://github.com/hpi-sam/digital-fuesim-manv</a
+            >
+            bereit. Weitere Informationen über das Projekt finden Sie auf der
```
Comment (anonym-HPI) [inline frontend/src/app/pages/landing-page/landing-page/landing-page.component.html:109]:
```suggestion
            zur Verfügung. Weitere Informationen über das Projekt finden Sie auf der
```

**Review by anonym-HPI — DISMISSED** [review]
not tested locally, only read

File: frontend/src/app/pages/landing-page/landing-page/landing-page.component.html (hunk 2 above)
Comment (Dassderdie) [reply frontend/src/app/pages/landing-page/landing-page/landing-page.component.html:101]:
```suggestion
            eines Massenanfalls von Verletzten (MANV).
```

File: README.md [hunk 5]
```diff
@@ -250,3 +250,20 @@ If you need to read from the state to change it, you should do this inside the a
 ## Licenses and Attributions
 
 -   License information about used images can be found [here](frontend/src/assets/image-sources.md). All images are licensed under their original license.
+
+## Contributors
```
Comment (ClFeSc) [inline README.md:254]: @christianzoellner @mbarkowsky Do you want to be mentioned here?

File: README.md [hunk 6]
```diff
@@ -250,3 +250,20 @@ If you need to read from the state to change it, you should do this inside the a
...
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<table>
+  <tr>
+    <td align="center"><a href="https://github.com/Dassderdie"><img src="https://avatars.githubusercontent.com/u/18506183?v=4" width="100px;" alt=""/><br /><sub><b>Julian Schmidt</b></sub></a></td>
+    <td align="center"><a href="https://github.com/ClFeSc"><img src="https://avatars.githubusercontent.com/u/68013019?v=4" width="100px;" alt=""/><br /><sub><b>Clemens Schielicke</b></sub></a></td>
+    <td align="center"><a href="https://github.com/hpistudent72"><img src="https://avatars.githubusercontent.com/u/64257074?v=4" width="100px;" alt=""/><br /><sub><b>Florian Krummrey</b></sub></a></td>
+    <td align="center"><a href="https://github.com/anonym-HPI"><img src="https://avatars.githubusercontent.com/u/68286419?v=4" width="100px;" alt=""/><br /><sub><b>Marvin Müller-Mettnau</b></sub></a></td>
+  </tr>
+</table>
```
Comment (Dassderdie) [inline README.md:267]: Add emoji keys: https://allcontributors.org/docs/en/emoji-key

File: README.md (hunk 5 above)
Comment (Dassderdie) [reply README.md:254]: I would mention them here. The emoji keys (see comment below) clarify their role in the project.

**Review by Dassderdie — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -250,3 +250,22 @@ If you need to read from the state to change it, you should do this inside the a
...
+    <td align="center"><a href="https://github.com/hpistudent72"><img src="https://avatars.githubusercontent.com/u/64257074?v=4" width="100px;" alt=""/><br /><sub><b>Florian Krummrey</b></sub></a><br /><a href="https://github.com/hpi-sam/digital-fuesim-manv/commits?author=hpistudent72" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/anonym-HPI"><img src="https://avatars.githubusercontent.com/u/68286419?v=4" width="100px;" alt=""/><br /><sub><b>Marvin Müller-Mettnau</b></sub></a><br /><a href="https://github.com/hpi-sam/digital-fuesim-manv/commits?author=anonym-HPI" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/mbarkowsky"><img src="https://avatars.githubusercontent.com/u/7481705?v=4" width="100px;" alt=""/><br /><sub><b>Matthias Barkowsky</b></sub></a><br />📆</td>
+    <td align="center"><a href="https://github.com/christianzoellner"><img src="https://avatars.githubusercontent.com/u/4678160?v=4" width="100px;" alt=""/><br /><sub><b>Christian Zöllner</b></sub></a><br />📆</td>
```
Comment (Dassderdie) [inline README.md:267]: I would at least also add the following emojis:
* @ClFeSc and @Dassderdie : `review`
* @anonym-HPI `platform` (deployment)

and maybe even a few keywords of the concrete parts the people worked on/better description of their role.

An example for me: 
`project-setup, frontend, synchronization, time-travel`

Comment (Dassderdie) [reply README.md]: I decided against the keywords for now.

The rest has been implemented with d7de69032c4971720c503deffdab6eddcfb81f79

**Review by Dassderdie — APPROVED** [review]
@ClFeSc Please have a look at d7de69032c4971720c503deffdab6eddcfb81f79 before merging.

**Comment by ClFeSc** [comment]
> @ClFeSc Please have a look at [d7de690](https://github.com/hpi-sam/digital-fuesim-manv/commit/d7de69032c4971720c503deffdab6eddcfb81f79) before merging.

Thanks!

**Pull request merged**